.clang-format

```diff
@@ -1,10 +1,17 @@
 ---
 Language:        Cpp
 BasedOnStyle:  Google
+ColumnLimit: 100
 TabWidth: 4
 IndentWidth: 4
 PointerAlignment: Left
 ReferenceAlignment: Pointer
 IndentAccessModifiers: false
 AccessModifierOffset: -4
+BinPackArguments: false
+BinPackParameters: false
+ExperimentalAutoDetectBinPacking: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortFunctionsOnASingleLine: Empty
+AllowShortLambdasOnASingleLine: Inline
```

Comment:
I don't have _any_ opinion about how C++ code should look like, and how much we want to customize compared to the base Google style (so happy to follow your preferences). But just to consider, s2geography just follows google (https://github.com/paleolimbot/s2geography/blob/master/.clang-format), and arrow uses only minimal overrides (https://github.com/apache/arrow/blob/main/.clang-format)

 (but so +1 on adding a linter for this to enforce consistent style!)

Reply:
I don't have strong opinions either and I think it is a good idea to stay close to the Google style used and not add too much customization. This is just to fix some minor things I find annoying, like packing some of the parameters on the same line when all the parameters don't fit on one line, e.g., I prefer this:

```cpp
m.def("equals",
      py::vectorize(Predicate(s2geog::s2_equals)),
      py::arg("a"),
      py::arg("b"));  
```

over this:

```cpp
m.def("equals", py::vectorize(Predicate(s2geog::s2_equals)), py::arg("a"),
      py::arg("b"));  
```

Reply:
As mentioned, I don't mind, so just follow your preferences ;)